.github/workflows/create-demo-registry.yml

```diff
@@ -0,0 +1,60 @@
+name: create-demo-registry
+on:
+  workflow_dispatch:
+  schedule:
```

Comment:
Instead of this hard coded workflow, can we generate this workflow using python readme.py file insde cli folder?

Reply:
The registry creation, validation, and deletion are different from other resources. That's why I manually added these two. I have excluded `registry` in `readme.py` so it won't create workflows for these.